pincer/oggparse.py

```diff
@@ -0,0 +1,96 @@
+"""
+This is a modified version of discord.py's OggParse module
+Copyright (c) 2015-2021 Rapptz
+Copyright (c) 2021-present Pincer
+:license: MIT, see LICENSE for details
+"""
+
+from __future__ import annotations
+
+import struct
+
+from typing import TYPE_CHECKING, ClassVar, IO, Generator, Tuple, Optional
+
+from .exceptions import PincerError
+
+__all__ = (
+    'OggError',
+    'OggPage',
+    'OggStream',
+)
+
+class OggError(PincerError):
```

Comment:
This should be in the exceptions file since it looks like the user has a way to catch it.